develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.md

```diff
@@ -6,6 +6,36 @@ uid: Protocol.Params.Param.Interprete
 
 Specifies how a parameter value is processed.
 
+By default, try to follow the following guidelines for Interpretes and Measurements:
+
+- **For strings**:
+
+  - Interprete.Rawtype = other
+
+  - Interprete.Type = string
+
+  - Measurement.Type = string
+
+- **For numbers**:
+
+  - Interprete.Rawtype = numeric text
+
+  - Interprete.Type = double
+
+  - Measurement.Type = number 
+
+- **For tables**:
+
+  - Interprete.Rawtype = other
```

Comment:
```suggestion
```
remove this line (for tables, the DIS snippets do not include an Interprete tag)